tests/improver-weighted-blending/02-basic_nonlin.bats

```diff
@@ -36,7 +36,7 @@
   KGO="weighted_blending/basic_nonlin/kgo.nc"
 
   # Run weighted blending with non linear weights and check it passes.
-  run improver weighted-blending 'nonlinear' 'forecast_reference_time' 'weighted_mean'\
```

Comment:
Having changed this line, it would be good to also change the string in line 34.
For all these CLI tests, please check and update the comment lines and the string in the "@test" line. Some of them have been copied from elsewhere and are incorrect.

Reply:
I'll do this but am not sure what's required in the "@test" line - it seems to vary a lot between tests.  Is there an accepted standard?